BUILD.md

```diff
@@ -34,40 +34,55 @@ below:
 | Apple Clang | 13.1.6
 | MSVC        | 19.23
 
-We do not recommend Windows for rippled production use at this time.
-As of January 2023, the Ubuntu platform has received the highest level of
+## Prerequisites
+
+See [System Requirements](https://xrpl.org/system-requirements.html).
+
+rippled generally requires:
+- git
+- Python
+- Conan
+- CMake
+- a C++ compiler
+
+### Linux
+
+The Ubuntu operating system has received the highest level of
 quality assurance, testing, and support.
-Additionally, 32-bit Windows development is not supported.
 
-Visual Studio 2022 is not yet supported.
-This is because rippled is not compatible with [Boost][] versions 1.78 or 1.79,
-but Conan cannot build Boost versions released earlier than them with VS 2022.
-We expect that rippled will be compatible with Boost 1.80, which should be
-released in August 2022.
-Until then, we advise Windows developers to use Visual Studio 2019.
+Here are [sample instructions for setting up a C++ development environment on Linux](./docs/build/environment.md#linux).
 
-[Boost]: https://www.boost.org/
+### Mac
 
+Many rippled engineers use macOS for development.
 
-## Prerequisites
+Here are [sample instructions for setting up a C++ development environment on macOS](./docs/build/environment.md#macos).
+
+### Windows
+
+Windows is not recommended for production use at this time.
 
-> **Warning**
-> These instructions assume you have a C++ development environment ready
-> with Git, Python, Conan, CMake, and a C++ compiler.
-> For help setting one up on Linux, macOS, or Windows,
-> please see [our guide](./docs/build/environment.md).
->
-> These instructions further assume a basic familiarity with Conan and CMake.
-> If you are unfamiliar with Conan,
-> then please read our [crash course](./docs/build/conan.md)
-> or the official [Getting Started][3] walkthrough.
+- Additionally, 32-bit Windows development is not supported.
+- Visual Studio 2022 is not yet supported.
+  - rippled generally requires [Boost][] 1.77, which Conan cannot build with VS 2022.
+  - Until rippled is updated for compatibility with later versions of Boost, Windows developers may need to use Visual Studio 2019.
+
+[Boost]: https://www.boost.org/
+
+## Conan profile
 
 To build this package, you will need Python (>= 3.7),
 [Conan][] (>= 1.55, < 2), and [CMake][] (>= 3.16).
 
 [Conan]: https://conan.io/downloads.html
 [CMake]: https://cmake.org/download/
 
+After you have a [C++ development environment](./docs/build/environment.md) ready with Git, Python, Conan, CMake, and a C++ compiler, you may need to set up your Conan profile.
```

Comment:
Can we surface this line up to the top of "Prerequisites" section? Because once I saw the dependencies and their versions, I downloaded it myself manually.
Since there are a few alternative ways to install these dependencies, it could cause discrepancy in the developer environment.

Reply:
I meant, bubble up the reference to the environment.md file

Reply:
Sure, although I think it shouldn't matter how you set up your environment. I believe there are multiple ways that should work.

Reply:
Added a link to the environment.md file near the top of "Prerequisites" - https://github.com/XRPLF/rippled/pull/4507/commits/27750c9e5f17b930546aa22386fe1552ed654d29